scss/_variables.scss

```diff
@@ -1838,4 +1838,15 @@ $step-link-marker-lg:    counter($stepped-process-counter) ".\A0" !default;
 $step-item-arrow-width:  1rem !default;
 $step-item-arrow-shape:  polygon(0% 0%, subtract(100%, $border-width) 50%, 0% 100%) #{"/* rtl:"} polygon(100% 0%, $border-width 50%, 100% 100%) #{"*/"} !default; // Used in clip-path
 // scss-docs-end stepped-process
+
+//// Sticker
+// scss-docs-start sticker
+$sticker-size-sm:                       $spacer * 7 !default;
+$sticker-size-md:                       $spacer * 9 !default;
+$sticker-size-lg:                       $spacer * 14 !default;
+
+$sticker-content-max-width-sm:          $sticker-size-sm * .7 !default;
+$sticker-content-max-width-md:          8.1875rem !default; // Should be $sticker-size-md * .7;
```

Comment:
- [ ] Check with the designers this behavior.

Reply:
FI, I changed the value from .7 to a more precise one using √2 / 2 -> .7071067812.

For the 3 sizes, the calculated width/height are:
```
scss/_variables.scss:1853 Debug: 6.1871843355rem
scss/_variables.scss:1854 Debug: 7.9549512885rem
scss/_variables.scss:1855 Debug: 12.374368671rem
```

If you draw the inner square:

<img width="150" alt="Capture d’écran 2022-01-10 à 08 08 26" src="https://user-images.githubusercontent.com/17381666/148730040-279656a9-c5a4-40f9-92d1-5cd149550750.png">